app/open_ai/text_events.py

```
@@ -0,0 +1,44 @@
+        raise ValueError(f"Output of gpt is None.\nig_text was:{ig_text}")
+    
+    try:
+        event_dict = json.loads(gpt_output)
+    except:
+        # TODO: Better error handling
+        logger.critical(f"Can't decode output of gpt into json.\ngpt_output: {gpt_output}",exc_info=True)
+        raise ValueError(f"Can't decode output of gpt into json.\ngpt_output: {gpt_output}")
+    
+    try:
+        event = EventCreate(**event_dict)
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)